app/notifications/process_notifications.py

```diff
@@ -50,18 +46,23 @@
 
 
 def create_content_for_notification(
-    template,
-    personalisation,
+    template: Template,
+    personalisation: dict,
 ):
-    template_object = get_template_instance(template.__dict__, personalisation)
-    check_placeholders(template_object)
+    """
+    Return an appropriate template instance from the notifications-utils repository,
+    or raise BadRequestError.
+    """
+
+    utils_template_instance = get_template_instance(template.__dict__, personalisation)
+    check_placeholders(utils_template_instance)
 
-    return template_object
+    return utils_template_instance
 
 
-def check_placeholders(template_object):
-    if template_object.missing_data:
-        message = 'Missing personalisation: {}'.format(', '.join(template_object.missing_data))
+def check_placeholders(utils_template_instance):
+    if utils_template_instance.missing_data:
+        message = 'Missing personalisation: {}'.format(', '.join(utils_template_instance.missing_data))
```

Comment:
Nit: since we are here, can we just convert this to f-string format?
```suggestion
        message = f"Missing personalisation: {', '.join(utils_template_instance.missing_data)}"
```

Reply:
In this case, given that the substitution is an expression and not just a variable, I think it's more readable with "format."